pkg/web/oauthclient/logout.go

```diff
@@ -59,20 +62,28 @@ func (oc *OAuthClient) HandleLogout(c echo.Context) error {
 	if logoutURL != "" {
 		_, tokenID, _, err := auth.SplitToken(token.AccessToken)
 		if err != nil {
-			return err
+			webhandlers.Error(w, r, err)
+			return
 		}
 		redirectURL := stripCommonRoot(logoutURL, config.RootURL)
 		query := url.Values{
 			"access_token_id":          []string{tokenID},
 			"post_logout_redirect_uri": []string{redirectURL},
 		}
 		u.RawQuery = query.Encode()
-		oc.removeAuthCookie(c)
-		return c.JSON(http.StatusOK, struct {
+		oc.removeAuthCookie(w, r)
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		enc := json.NewEncoder(w)
+		enc.SetIndent("", "  ")
```

Comment:
I don't think it makes much sense to have indentation, but good catch on the content type header.